src/utils/uploader.util.js

```diff
@@ -0,0 +1,23 @@
+import config from '../config';
+
+const { CLOUD_NAME, UPLOAD_PRESET } = config;
+
+export default (upload) => {
```

Comment:
Why not use axios for this?

Reply:
Hi @Arthur236,

About this, the editor we are using uses the event we are dispatching in the `xhr.onreadystate` to update the progress of the upload (line 11).

Reply:
Alright 👍